src/async-wrap.cc

```diff
@@ -201,42 +203,49 @@ Local<Value> AsyncWrap::MakeCallback(const Local<Function> cb,
     Local<Value> enter_v = domain->Get(env()->enter_string());
     if (enter_v->IsFunction()) {
       enter_v.As<Function>()->Call(domain, 0, nullptr);
-      if (try_catch.HasCaught())
+      if (try_catch.HasCaught()) {
+        FatalException(env()->isolate(), try_catch);
```

Comment:
If `try_catch.SetVerbose(true)` was called, wouldn't `FatalException` be called anyway?